rosetta/services/construction.go

```diff
@@ -19,7 +19,7 @@ import (
 
 const (
 	// DefaultGasPrice ..
-	DefaultGasPrice = denominations.Nano
+	DefaultGasPrice = 40 * denominations.Nano
```

Comment:
the default is 30, right?  why change to 40?

Reply:
default is 1